stack-mac-8.6.1.yaml

```diff
@@ -10,3 +10,30 @@ resolver: ghc-8.6.1
 compiler: ghc-8.6.1
 compiler-check: match-exact
 packages: []
+
+allow-newer: true
```

Comment:
I think, if you're on it, you can use latest Stackage Nightly snapshot since it's already on Stackage. And then rename `stack-mac.yaml` file to just `stack.yaml`.

* https://www.stackage.org/nightly-2018-10-01